kotlin-example/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,7 +3,3 @@ distributionPath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-
```

Comment:
why removed?

Reply:
This was part of the wrapper autogenerated changes.

Reply:
add it back please.. we use this for minification testing & validation on the kotlin-example app

Reply:
Moved to the proper spot in gradle.properties.

R8 is now enabled by default for both app and Android library projects using Android Gradle plugin 3.4.0 and higher.